apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLoggerMiddleware.cs

```diff
@@ -27,15 +28,16 @@ namespace Google.Cloud.Diagnostics.AspNetCore
     public sealed class ErrorReportingExceptionLoggerMiddleware
```

Comment:
I was trying to think if this should be renamed as well, but `ErrorLoggingExceptionLoggerMiddleware` is quite redundant. However, `Error/Exception` and `Reporting/Logger` already seems redundant. What about `ExceptionLoggingMiddleware`?